mxcubecore/utils/units.py

```diff
@@ -0,0 +1,71 @@
+"""
+Utility functions for converting between different units.
+"""
+
+#
+# time units
+#
+
+
+def us_to_sec(us) -> float:
```

Comment:
```suggestion
def us_to_sec(us: float) -> float:
```

For all the others as well. Note that the type hint `float` will also accept `int`.

Reply:
Good catch, added parameter type annotations.